pandas-stubs/core/indexes/accessors.pyi

```diff
@@ -154,40 +154,40 @@ class _DatetimeLikeOps(
 # type of the series, we don't know which kind of series was ...ed
 # in to the dt accessor
 
-_DTRoundingMethodReturnType = TypeVar(
-    "_DTRoundingMethodReturnType",
+_DTTimestampTimedeltaReturnType = TypeVar(
+    "_DTTimestampTimedeltaReturnType",
```

Comment:
Rename to a general name to reuse this type var in the return type annotation for `as_unit` method